### PR TITLE
Update Confetti.svelte

### PR DESCRIPTION
**Pull request by mstachowiak**
Close statements

- - -
This pull request can be reviewed on [GitContext.com](https://gitcontext.com/app/prs/github/mstachowiak/svelte-confetti/2)

### CODE REVIEW

File: src/lib/Confetti.svelte [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <script>
   import { onMount } from "svelte"
 
-  export let size = 10
-  export let x = [-0.5, 0.5]
-  export let y = [0.25, 1]
+  export let size = 10;
```
Comment (mikemntlabs) [inline src/lib/Confetti.svelte:4]: Looking good, thanks for doing this.

File: src/lib/Confetti.svelte [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+  export let x = [-0.5, 0.5];
+  export let y = [0.25, 1];
   export let duration = 2000
```
Comment (mikemntlabs) [inline src/lib/Confetti.svelte:7]: Did you consider doing these as well?

**Review by mikemntlabs — CHANGES_REQUESTED** [review]
Reviewed on GitContext: 0/1 files reviewed, 1 blocking issues

**Review by mikemntlabs — CHANGES_REQUESTED** [review]
Reviewed on GitContext: 1/1 files reviewed, 1 blocking issues

File: src/lib/Confetti.svelte (hunk 2 above)
Comment (mstachowiak) [reply src/lib/Confetti.svelte:7]: I haven't, should I?

**Review by mstachowiak — COMMENTED** [review]
Reviewed on GitContext: 0/1 files reviewed, 0 blocking issues

File: src/lib/Confetti.svelte (hunk 1 above)
Comment (mstachowiak) [reply src/lib/Confetti.svelte:4]: No problem

File: src/lib/Confetti.svelte (hunk 2 above)
Comment (mstachowiak) [reply src/lib/Confetti.svelte:7]: How about now?

**Review by mstachowiak — COMMENTED** [review]
Reviewed on GitContext: 0/1 files reviewed, 0 blocking issues

File: src/lib/Confetti.svelte (hunk 1 above)
Comment (mikemntlabs) [reply src/lib/Confetti.svelte:4]: You sure?

**Review by mikemntlabs — CHANGES_REQUESTED** [review]
Reviewed on GitContext: 1/1 files reviewed, 1 blocking issues

**Review by mstachowiak — COMMENTED** [review]
Reviewed on GitContext: 0/1 files reviewed, 0 blocking issues

File: src/lib/Confetti.svelte (hunk 1 above)
Comment (mstachowiak) [reply src/lib/Confetti.svelte:4]: Yep

**Review by mikemntlabs — CHANGES_REQUESTED** [review]
Reviewed on GitContext: 1/1 files reviewed, 1 blocking issues

File: src/lib/Confetti.svelte (hunk 1 above)
Comment (mikemntlabs) [reply src/lib/Confetti.svelte:4]: Again!

**Review by mstachowiak — COMMENTED** [review]
Reviewed on GitContext: 0/1 files reviewed, 0 blocking issues

File: src/lib/Confetti.svelte (hunk 1 above)
Comment (mstachowiak) [reply src/lib/Confetti.svelte:4]: Acknowledged

**Review by mikemntlabs — CHANGES_REQUESTED** [review]
Reviewed on GitContext: 1/1 files reviewed, 1 blocking issues

File: src/lib/Confetti.svelte (hunk 1 above)
Comment (mikemntlabs) [reply src/lib/Confetti.svelte:4]: The published objects weren't stateful!

**Review by mstachowiak — COMMENTED** [review]
Reviewed on GitContext: 0/1 files reviewed, 0 blocking issues

File: src/lib/Confetti.svelte (hunk 1 above)
Comment (mstachowiak) [reply src/lib/Confetti.svelte:4]: Are they stateful now?

Comment (mikemntlabs) [reply src/lib/Confetti.svelte:4]: Yeah, fixed!  The only challenge now is that `refresh` doesn't update the thread state for some reason.

**Review by mikemntlabs — CHANGES_REQUESTED** [review]
Reviewed on GitContext: 1/1 files reviewed, 1 blocking issues

**Review by mstachowiak — COMMENTED** [review]
Reviewed on GitContext: 0/1 files reviewed, 0 blocking issues

**Review by mikemntlabs — APPROVED** [review]
Reviewed on GitContext: 1/1 files reviewed, 0 blocking issues

**Pull request merged**